rocks-db/src/migrations/offchain_data.rs

```diff
@@ -0,0 +1,64 @@
+use bincode::deserialize;
+use rocksdb::MergeOperands;
+use tracing::error;
+
+use crate::columns::offchain_data::{OffChainData, OffChainDataDeprecated, StorageMutability};
+use crate::migrator::{RocksMigration, SerializationType};
+use crate::ToFlatbuffersConverter;
+
+impl From<OffChainDataDeprecated> for OffChainData {
+    fn from(value: OffChainDataDeprecated) -> Self {
+        let immutability = StorageMutability::from(value.url.as_str());
+        Self {
+            storage_mutability: immutability,
+            url: Some(value.url),
+            metadata: Some(value.metadata),
+            last_read_at: 0,
+        }
+    }
+}
+
+impl OffChainData {
```

Comment:
Why is it implemented on the new OffChainData, while it operates on the OffChainDataDeprecated?

Reply:
And it's also putting the data back in the flatbuffer format. This seems wrong in terms of the merge function